snare.py

```diff
@@ -73,7 +74,7 @@ def __init__(self, meta, run_args, debug=False, keep_alive=75, **kwargs):
                         print(e)
                     finally:
                         r.release()
-        except:
+        except Exception:
```

Comment:
it will be really good, if we change Exception here to specific aiohttp exception. Because using broader Exception is very bad :)